packages/cli/src/lib/setup/setupList.ts

```diff
@@ -33,13 +33,19 @@ interface FlagObject {
     ip?: string;
 }
 
+interface ListOptions {
```

Comment:
Just a nit: When I started reworking some of the setup classes to CLIxyz, part of the intent was to make it clear these are CLI commands.
I think it would be good to reflect this in the options interface name, e.g.
```suggestion
interface CLIListOptions {
```
In the long run we could then merge the CLI and setup variants of the implementations and have them named consistently.